### ISSUE TITLE
Which trainset of Valentini dataset you use for training?

### ISSUE THREAD

**Issue by Kelvin945** [facebookresearch/denoiser#29]
Great work, I was trying to reproduce your result, but there are two trainsets in Valentini dataset, one with 56 speakers, the other contains 28 speakers. Could you let me know which train set you are used for training?

**Comment by adiyoss** [comment]
Hi,
The results reported in the paper were obtained using the 28 speakers version, however, you will get better performance using the 56 speakers version.

**Comment by Kelvin945** [comment]
Thank you, in other issues, you mention that 

> For validation, we sampled mixture data from two other speakers in the training set.

Did you mean the validation set is a subset of the training set or they are exclusive?

Also, what would be the purpose of the validation set here?

**Comment by adiyoss** [comment]
We excluded two speakers from the training set and used them for validation, that way we can evaluate the model on unseen speakers.
We use the validation set to avoid overfitting by saving the model with the best loss value on the validation set.

**Comment by amieroten** [comment]
Hi there, I am also looking to reproduce your results using the Valentini dataset. Do you recall which particular speakers you left out from training to use as the validation set?

**Comment by gujralsanyam22** [comment]
we use validation dataset on this we have train and test the model then we see how the result comes closer and good

**Comment by adiyoss** [comment]
Hi @amieroten,
We use speakers p286 and p287 for the validation set.

**Comment by adiyoss** [comment]
Closing this issue. In case you have more questions feel free to reopen.
